config/ember-try.js

```diff
@@ -29,22 +29,22 @@ module.exports = async function () {
           },
         },
       },
-      {
-        name: 'ember-beta',
-        npm: {
-          devDependencies: {
-            'ember-source': await getChannelURL('beta'),
-          },
-        },
-      },
-      {
-        name: 'ember-canary',
-        npm: {
-          devDependencies: {
-            'ember-source': await getChannelURL('canary'),
-          },
-        },
-      },
+      // {
```

Comment:
create an issue for re-enabling

Reply:
https://github.com/denali-design/denali-ember/issues/85